share/revng/test/tests/analysis/ImportFromCAnalysis/rft-with-an-unknown-register/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Unsupported register location.
```

Comment:
`Unsupported register location in _REG annotation.`

BONUS

`Valid values are: ` and then you print the valid values we accept for the given ABI

Reply:
The latter shouldn't be part of the test though. It makes it much more brittle.

Reply:
I disagree that we should encode all the register names in here. But even if we do, that needs to be more systematic and done for all the annotations. And some of them (say abi) are not going to be capped to a specific list (since adding an abi is, eventually, going to be as easy as adding a new file to a specified directory). Which makes it even more awkward. But I guess we can further discuss this as the part the error clean up task `@aleclearmind` scheduled for this.

Reply:
It's true that when we support user-defined ABIs it's gonna be harder to enumerate the list of valid registers for each ABI, but I expect that to be perfectly doable at runtime. If there's a user-defined ABI, that will be picked up and will be dynamically available to this parsing routines too.

Anyway, let's leave the register names out of the picture for now. We'll deal with them when we properly rehaul error messages in the future.